yt/yt/ytlib/chunk_client/chunk_fragment_reader.cpp

```diff
@@ -764,9 +764,11 @@ class TChunkFragmentReader::TPeriodicProbingSession
 
     static TClientChunkReadOptions MakeOptions()
     {
+        auto workloadDescriptor = TWorkloadDescriptor(EWorkloadCategory::UserBatch);
+        workloadDescriptor.DiskFairShareBucketTag = "TPeriodicProbingSession";
```

Comment:
Are these options used only for probing?

Reply:
I don't know. I think it should be, because it is a static method inside `class TChunkFragmentReader::TPeriodicProbingSession`